src/AcceptanceTesting/Common/ProviderPayments.TestStack.UI/ProviderPayments.TestStack.UI.csproj

```diff
@@ -210,11 +210,6 @@
     <Compile Include="Controllers\PaymentReportController.cs" />
     <Compile Include="Controllers\ProcessController.cs" />
     <Compile Include="Controllers\ComponentAdminController.cs" />
-    <Compile Include="DependencyResolution\ControllerConvention.cs" />
```

Comment:
Are these removals correct?

Reply:
The files are not present in the repo at the location specified in the csprof file, which causes the build to fail. They appear to have been moved to a new location.